packages/synapse-types/src/File/FileHandle.ts

```diff
@@ -27,7 +27,7 @@ export interface FileHandle {
   /** The short, user visible name for this file. */
   fileName: string
   /** The optional storage location descriptor */
-  storageLocationId: number
+  storageLocationId: number | null
```

Comment:
Update type to handle case where storage location id is null when storage location field was not back-filled, as per comment in FileHandleUtils

Reply:
It's definitely sometimes undefined, not sure if it is ever `null`, so you may consider changing it to this. But if this works, I'm ok with the type imprecision; we may be moving to generated models soon, so I don't think it's that important that this is perfect for nullish values
```suggestion
  storageLocationId?: number
```

Reply:
For now, I'll include `null` in the type since the [previous implementation](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/045b8597f9a12ee96b76da55e768eddd64740a5d/packages/synapse-react-client/src/utils/functions/FileHandleUtils.ts#L87) checked for null storage location ids and I maintained the null check in the [PR changes](https://github.com/hallieswan/synapse-web-monorepo/blob/0e0ad995e3b5361123882d311b314c07d4e0ee84/packages/synapse-react-client/src/utils/functions/FileHandleUtils.ts#L88), but happy to change in the future!